compose/compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+  mysql:
+    image: mysql:8.0.33
+    ports: [ '8033:3306' ]
+    # to supress mbind: Operation not permitted in CI
+    # https://stackoverflow.com/a/55706057
+    cap_add:
+      - SYS_NICE
+    environment:
+      MYSQL_ROOT_PASSWORD: msandbox
+    healthcheck:
+      test: mysqladmin ping -h 127.0.0.1 -u root --password=$$MYSQL_ROOT_PASSWORD
+      start_period: 1s
+      interval: 1s
+      timeout: 2s
+      retries: 60
+    volumes:
+      - mysql-standalone-data-dir:/var/lib/mysql
+      - ./bootstrap.sql:/docker-entrypoint-initdb.d/bootstrap.sql
+
+  test:
+    build:
+      context: ../
+    command: go test -race -v ./...
+    depends_on:
+      mysql:
+        condition: service_healthy
+    environment:
+      MYSQL_DSN: root:msandbox@tcp(mysql)/test
```

Comment:
[Tests require `root`](https://github.com/cashapp/spirit/blob/788655ae00e1e320bc75b964cea337629f13c545/pkg/check/privileges_test.go#L19). This works with the `msandbox` user too because the password happens to be the same.